syntax = "proto3";
import "google/protobuf/timestamp.proto";

message Punch { bytes raw = 1; }

message Punches {
  repeated Punch punches = 1;
  google.protobuf.Timestamp sending_timestamp = 2;
}
